### PR TITLE
catalog is ready

### PR DESCRIPTION
**Pull request by lytvyshko**
- [DEMO LINK](https://lytvyshko.github.io/layout_catalog_grid/)

### CODE REVIEW

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
+  justify-content: center;
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 200px;
+    display: grid;
+    place-items: center;
+    width: $card-width;
```
Comment (rostyslav-pyliak) [inline src/styles/_catalog.scss:11]: width is redundant here

**Review by rostyslav-pyliak — APPROVED** [review]
Good job

File: src/styles/_catalog.scss (hunk 1 above)
Comment (lytvyshko) [reply src/styles/_catalog.scss:11]: it's necessary for viewport < 488px

Comment (rostyslav-pyliak) [reply src/styles/_catalog.scss:11]: Ok. Now I see it. Sorry)

Comment (Denchepornyuk) [reply src/styles/_catalog.scss:11]: if you add `grid-template-columns: repeat(1, $card-width);` to .catalog than you can remove width

**Review by denys-cheporniuk — APPROVED** [review]

**Pull request closed without merging**